types/akamai-edgeworkers/test/scenario-crypto.ts

```diff
@@ -116,13 +133,86 @@ KlSNHLY0ZX554kjI8DknO3x8J5z+H31OX7spkrI6xdqj9Q0Ouoy6UmjJ3w==
         });
 
     /**
-     * A function that allows you to get cryptographically strong random values
-     * @param typeArray: An integer-based TypedArray
+     * Imports the key
+     * @param format string describing the data format of the key to import
+     * @param keyData An ArrayBuffer, a TypedArray, a DataView, or a JSONWebKey object containing the key
+     * @param algorithm An object defining the type of key to import
+     * @param extractable A boolean value indicating whether it will be possible to export the key
+     * @param keyUsages An array indicating the operations that can be done with the key
+     *
+     * @returns A promise that fulfills with the imported key as a CryptoKey object.
+     */
+    const hmac_imported_key = await crypto.subtle.importKey(
+        "jwk",
+        {
+            kty: "oct",
+            k: "Y0zt37HgOx-BY7SQjYVmrqhPkO44Ii2Jcb9yydUDPfE",
+            alg: "HS256",
+            ext: false
+        },
+        {
+            name: "HMAC",
+            hash: "SHA-256"
+        },
+        false,
+        ["sign", "verify"]
+    );
+    const data = Uint8Array.from([97, 110, 103, 117, 115, 32, 97, 110, 100, 32, 111, 119, 101, 110]);
+
+    /**
+     * Sign generates a digital signature.
+     * @param algorithm A string or object that specifies the signature algorithm to use and its parameters
+     * @param key A CryptoKey object containing the key to be used for signing
+     * @param data An ArrayBuffer, a TypedArray or a DataView object containing the data to be signed
+     *
+     * @returns A Promise that fulfills with an ArrayBuffer containing the signature
+     */
+    const sig = await crypto.subtle.sign("HMAC", hmac_imported_key, data);
+    /**
+     * Verify a digital signature
+     * @param algorithm An object specifying the algorithm to be used
+     * @param key A CryptoKey containing the key that will be used to verify the signature
+     * @param signature ArrayBuffer containing the signature to verify
+     * @param data ArrayBuffer containing the data whose signature is to be verified
+     *
+     * @returns A promise that fulfills with a boolean value: true if the signature is valid, false otherwise
+     */
+    await crypto.subtle.verify("HMAC", hmac_imported_key, sig, data);
```

Comment:
Shouldn't we assert that the verification succeeded? IIRC `verify()` returns a boolean so we should assert on that somehow.